### PR TITLE
breaking: prom-client to v13

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

The new version of the prom-client v13 contains some breaking changes we have to account for.

CleanShot 2020-12-21 at 12 30 22@2x

### CODE REVIEW

**Review by tdeekens — COMMENTED** [review]
@emmenko would you mind reviewing this?

File: .changeset/nasty-horses-enjoy.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+"@promster/metrics": patch
+"@promster/server": patch
+"@promster/types": patch
+---
+
+# Breaking Changes
```
Comment (emmenko) [inline .changeset/nasty-horses-enjoy.md:7]: It should be `major` right?
I think changeset bumps a major anyway when peer deps change

File: readme.md [hunk 2]
```diff
@@ -294,11 +294,11 @@ You can use the `express` or `hapi` package to expose the gathered metrics throu
 const app = require('./your-express-app');
 const { getSummary, getContentType } = require('@promster/express');
 
-app.use('/metrics', (req, res) => {
+app.use('/metrics', async (req, res) => {
   req.statusCode = 200;
 
   res.setHeader('Content-Type', getContentType());
-  res.end(getSummary());
+  res.end(async getSummary());
```
Comment (emmenko) [inline readme.md:301]:
```suggestion
  res.end(await getSummary());
```

**Review by emmenko — COMMENTED** [review]
Looks good

File: .changeset/nasty-horses-enjoy.md (hunk 1 above)
Comment (tdeekens) [reply .changeset/nasty-horses-enjoy.md]: Thanks.

**Pull request merged**